azure-pipelines.yml

```diff
@@ -7,6 +7,8 @@ variables:
   # The $(FlutterToolPath) variable is generated by the FlutterInstall task 
   # See code in the Flutter Build task https://github.com/aloisdeniel/vsts-flutter-tasks/blob/master/tasks/build/index.ts
   flutterExecPath: $(FlutterToolPath)/flutter.bat
+  versionGetRegex: '(?<=version: ).*'
```

Comment:
I would probably drop `Get` from the name and potentially rename it to `versionNumberRegex` - feel free to adjust accordingly (i.e. if it's not actually a number)

Reply:
Updated to your preference ;)

Reply:
Hehehe, thanks. I know I can be very peculiar about naming of variables, classes etc., which may seem very picky - I apologise. The real reason behind my suggestions for changes in such cases is the fact that naming things in code is very important. Even though you think it might be irrelevant, the readability of code is a key factor to a good and clean code. Also it's really hard, to name things appropriately.

Regardless, be sure to call out things that you don't agree with (in comments) and/or you have better suggestions. I'm also not always right and my suggestions might not align with what the code represents etc., so make sure to call these out.